package.json

```diff
@@ -4,15 +4,15 @@
   "description": "`BinderHub`",
   "devDependencies": {
     "bootstrap": "^3.4.0",
-    "clipboard": "^1.7.1",
-    "css-loader": "^0.28.7",
-    "event-source-polyfill": "^0.0.12",
-    "extract-text-webpack-plugin": "^3.0.2",
-    "file-loader": "^1.1.6",
```

Comment:
"file-loader"  is also deprecated (see https://webpack.js.org/loaders/file-loader/). [asset/resource](https://webpack.js.org/guides/asset-modules/) is used instead.